### PR TITLE
Chore: Update README, dotfiles, license, version, linting.

### PR DESCRIPTION
**Pull request by JoeBello**

### CODE REVIEW

File: src/util.js [hunk 1]
```diff
@@ -6,23 +6,14 @@
 plugin.service('wgnUtil', [
 	function() {
 
-		/**
-		 * Service
-		 *
-		 * @type {Object}
-		 */
-		var Service = this;
```
Comment (alexweber) [inline src/util.js:14]: @JoeBello hey man I'm actually gonna fight ya on this one... it's 100% a best practice to not use `this` directly in angular services, controllers, etc.  You really want to be doing:
```js
plugin.service('wgnUtil', [function () {
  var srv = this;

  srv.foo = 'whatever';

  srv.bar = function (param) { alert() };

  return srv;
}]);
```

File: src/util.js [hunk 2]
```diff
@@ -6,23 +6,14 @@
...
 		 *
 		 * @return {String} UUID
 		 */
 		this.getUUID = function() {
-			return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function(c) {
-				var r = Math.random() * 16 | 0, v = c == 'x' ? r : (r & 0x3 | 0x8);
-				return v.toString(16);
-			});
+		 return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g,
+		   function(c) { return parseInt(Math.random() * 16).toString(16); });
```
Comment (alexweber) [inline src/util.js]: @JoeBello that's an interesting change you made there, can you plz just explain the motivation?

File: src/util.js [hunk 3]
```diff
@@ -65,19 +56,19 @@ plugin.service('wgnUtil', [
...
 
             var params = (keyValue || "").split('&');
-            
+
 			params.forEach(function(keyValue) {
 				if (keyValue) {
 					key_value = keyValue.split('=');
 					key = decodeURIComponent(key_value[0]);
-					obj[key] = (key_value[1] !== undefined)? decodeURIComponent(key_value[1]) : true;
+					obj[key] = key_value[1] !== undefined ?
```
Comment (alexweber) [inline src/util.js:65]: @JoeBello no reason to break lines here i dont think

**Comment by alexweber** [comment]
@JoeBello almost there man!

**Comment by alexweber** [comment]
@JoeBello also dont be closing your own PRs like that you get yourself in trouble ;)
if you push more commits to your branch it will update the PR with the changes so you're all set

File: src/util.js (hunk 1 above)
Comment (JoeBello) [reply src/util.js:14]: I can't argue this one.

File: src/util.js (hunk 2 above)
Comment (JoeBello) [reply src/util.js]: Linting failed due to use of the bitwise operators and the double equal.

File: src/util.js (hunk 3 above)
Comment (JoeBello) [reply src/util.js:65]: Total single line length is 86 characters.

Comment (alexweber) [reply src/util.js:65]: ok fair enough :)

File: src/util.js (hunk 2 above)
Comment (alexweber) [reply src/util.js]: have you compared the outputs and stuff?  sure that removing the bitwise is not causing any other unintended consequences?
in this case the linting rule was most likely arbitrary so we can be flexible and change it

**Pull request merged**